tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3863,3 +3863,806 @@ Feature: resources shared by user
         }
       }
       """
+
+  @env-config
+  Scenario: sharer lists file share with role Secure Viewer after share role is disabled (Personal Space)
+    Given the administrator has enabled the permissions role "Secure Viewer"
+    And user "Alice" has uploaded file with content "some content" to "textfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt  |
+      | space           | Personal      |
+      | sharee          | Brian         |
+      | shareType       | user          |
+      | permissionsRole | Secure Viewer |
+    And the administrator has disabled the permissions role "Secure Viewer"
+    When user "Alice" lists the shares shared by her using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should contain resource "textfile.txt" with the following data:
+      """
+      {
+        "type": "object",
+        "required": [
+          "parentReference",
+          "permissions",
+          "name",
+          "size"
+        ],
+        "properties": {
+          "parentReference": {
+            "type": "object",
+            "required": [
+              "driveId",
+              "driveType",
+              "path",
+              "name",
+              "id"
+            ],
+            "properties": {
+              "driveId": {
+                "type": "string",
+                "pattern": "^%space_id_pattern%$"
+              },
+              "driveType": {
+                "const": "personal"
+              },
```

Comment:
Ca we try this format for property checks? :thinking: should be fine and it reduces the lines
```suggestion
              "driveType": { "const": "personal" },
```

Reply:
Yes we can do this where there is single property value to check. :+1: